api/courses/index.js

```diff
@@ -0,0 +1,29 @@
+const router = require("express").Router();
+const createCourse = require("../../helpers/Courses/createCourse");
+const loginMiddleware = require("../../middleware/checkLoggedIn");
+const searchQuery = require("../../helpers/Courses/searchQuery");
+
+router.post("/", loginMiddleware, async(req, res) => {
+  let { courseName, initialSessionPrice, sessionHourlyRate } = req.body;
+  if(!courseName || !initialSessionPrice || !sessionHourlyRate) {
+    return res.sendStatus(400);
+  }
+  await createCourse(req.user.id, courseName, initialSessionPrice, sessionHourlyRate);
+  res.end();
+});
+
+//search
```

Comment:
remove this comment `GET` is self documenting

Reply:
c4f363c71f87c14edcda4ccab29f044dfc6d3963